dnf/base.py

```diff
@@ -665,6 +669,7 @@ def do_transaction(self, display=()):
 
         if not self.transaction:
             if self._group_persistor:
+                self._trans_success = True
```

Comment:
dnf uses:
_finalize_comps_trans()
_trans_tempfiles
self._comps_trans
... 
therefore I would like to keep it like it is